workflow/rules/report.smk

```diff
@@ -1,21 +1,20 @@
-rule igv_report:
+rule vcf_report:
     input:
-        bcf="results/merged-calls/{group}.{event}.fdr-controlled.bcf",
         ref="resources/genome.fasta",
-        ref_idx="resources/genome.fasta.fai",
-        bams=lambda w: get_group_bams(w),
-        bais=lambda w: get_group_bams(w, bai=True)
+        bams=lambda w: get_batch_bams(w),
+        bcfs=lambda w: expand("results/merged-calls/{group}.{{event}}.fdr-controlled.bcf", group=get_report_batch(w))
     output:
-        report("results/igv-report/{group}.{event}.html", caption="../report/calls.rst", category="Variant calls")
-    log:
-        "logs/igv-report/{group}.{event}.log"
+        report(directory("results/vcf-report/{batch}.{event}/"), htmlindex="index.html", caption="../report/calls.rst", category="Variant calls")
     params:
-        os.path.join(os.path.dirname(workflow.snakefile), "resources/igv-report-template.html")
+        bcfs=lambda w: expand("{group}=results/merged-calls/{group}.{event}.fdr-controlled.bcf", group=get_report_batch(w), event=w.event),
+        bams=lambda w: get_batch_bams(w, True),
+        format_field = "DP AF OBS",
+        template = Path(Path(workflow.snakefile).parent, "resources/custom-table-report.js"),
+        max_read_depth = config["report"]["max_read_depth"],
+        js_files = Path(Path(workflow.snakefile).parent, "resources/math.min.js")
+    log:
+        "logs/igv-report/{batch}.{event}.log"
     conda:
-        "../envs/igv-reports.yaml"
+        "../envs/rbt.yaml"
     shell:
-        "bcftools view {input.bcf} > {output}.vcf;"
-        "create_report {output}.vcf {input.ref} --flanking 100 "
-        "--info-columns ANN dgiDB_drugs cosmic_LEGACY_ID --info-columns-prefixes PROB_ --sample-columns DP AF OBS"
-        " --template {params} --tracks {input.bams} --output {output} --standalone 2>&1 > {log}; "
-        "rm {output}.vcf"
+        "rbt vcf-report {input.ref} --bams {params.bams} --vcfs {params.bcfs} -f {params.format_field} -i PROB_* -j {params.template} -d {params.max_read_depth} -l {params.js_files} -- {output}"
```

Comment:
```suggestion
        "rbt vcf-report {input.ref} --bams {params.bams} --vcfs {params.bcfs} -f {params.format_field} "
        "-i PROB_* -j {params.template} -d {params.max_read_depth} -l {params.js_files} -- {output}"
```

Reply:
Further, please always use the long option names instead of the short ones (readability).

Reply:
Looks to me as this suggestion hasn't been applied.

Reply:
The one with the two-liner.

Reply:
Wait I'll do it.